packages/tools/src/types/IToolGroup.ts

```diff
@@ -19,8 +20,13 @@ export default interface IToolGroup {
   getViewportsInfo: () => Array<Types.IViewportId>;
   /** Get the toolInstance of the toolName */
   getToolInstance: { (toolName: string): any };
-  /** Add a tool to toolGroup with its configuration */
-  addTool: { (toolName: string, toolConfiguration?: any): void };
+  /** Add a tool to toolGroup with its configuration and custom calculator if wanted */
+  addTool: {
+    (
+      toolName: string,
+      toolConfiguration?: Record<any, any> & { statsCalculator?: Calculator }
```

Comment:
This should be ToolConfiguration

Reply:
Oh true , I forgot about it. I'll update it now